src/DynamoCore/Library/LibraryServices.cs

```diff
@@ -816,12 +816,14 @@ private void ImportProcedure(string library, ProcedureNode proc)
                             defaultArgumentNode = binaryExpr.RightNode;
                         }
                     }
+                    string shortName = null;
                     if (defaultArgumentNode != null)
                     {
+                        shortName = defaultArgumentNode.ToString();
```

Comment:
Let's not assume that defaultArgumentNode sent here will always have short names.